jobs/server.go

```diff
@@ -20,11 +20,11 @@ type JobServer struct {
 	DataRetentionJob        ejobs.DataRetentionJobInterface
 	MessageExportJob        ejobs.MessageExportJobInterface
 	ElasticsearchAggregator ejobs.ElasticsearchAggregatorInterface
-	ElasticsearchIndexer    ejobs.ElasticsearchIndexerInterface
+	ElasticsearchIndexer    tjobs.IndexerJobInterface
 	LdapSync                ejobs.LdapSyncInterface
 	Migrations              tjobs.MigrationsJobInterface
 	Plugins                 tjobs.PluginsJobInterface
-	BleveIndexer            tjobs.BleveIndexerInterface
+	BleveIndexer            tjobs.IndexerJobInterface
```

Comment:
what do you think of removing the t from tjobs and only leave the enterprise with the suffix?

Reply:
Makes sense now that we have interfaces that qualify for both team and enterprise implementations, but sadly causes naming collisions:

```go
"github.com/mattermost/mattermost-server/v5/jobs"
tjobs "github.com/mattermost/mattermost-server/v5/jobs/interfaces"
```

I would leave it as it is for now